src/tests/conftest.py

```diff
@@ -741,3 +767,17 @@ def ggwgebieden_data(ggwgebieden_model):
     ggwgebieden_model.bestaat_uit_buurten.through.objects.create(
         ggwgebieden_id="03630950000000.1", bestaat_uit_buurten_id="03630000000078.1"
     )
+
+
+@pytest.fixture()
+def woningbouwplannen_data(woningbouwplan_model):
+    woningbouwplan_model.objects.create(
+        id="1",
```

Comment:
Als je hier de comma weglaat zal black het wel op 1 regel zetten.

Reply:
Ja, korter is in dit geval beter denk ik.